homeassistant/components/tessie/strings.json

```diff
@@ -315,5 +315,22 @@
         "name": "[%key:component::update::title%]"
       }
     }
+  },
+  "exceptions": {
+    "not_supported": {
+      "message": "{name} is not supported."
+    },
+    "cable_connected": {
+      "message": "Charge cable is connected."
```

Comment:
```suggestion
      "message": "Charge cable is connected to {name}."
```

Reply:
Name is the entity not the vehicle.

This would result in "charge cable connected to chart port door".